src/app/pages/loading-states/loading-states.component.html

```diff
@@ -0,0 +1,54 @@
+<mat-toolbar color="primary" flexLayout="row">
+    <button *ngIf="isSmall" mat-icon-button (click)="toggleMenu()" style="margin-left: -8px;">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <h6>Loading States</h6>
+    <div fxFlex></div>
+    <button mat-button color="white" (click)="refreshData()">
```

Comment:
In the other examples with toolbar actions, I believe we are using icon buttons instead of normal buttons...do we want to be consistent?

Reply:
Good catch, I made this a mat-icon-button